### PR TITLE
Add listing and landing page for JFrog webinar

### PR DESCRIPTION
**Pull request by zchase**
This PR adds the JFrog webinar to the webinar listing page and the superpower page. I created an image for this, but feel free to replace it if we have a better one.

Fixes: https://github.com/pulumi/docs/issues/3112

### CODE REVIEW

File: content/webinars/sharing-and-reusing-infrastructure/index.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        - name: Melissa McKay
+          role: Developer Advocate, JFrog
+
+    # A bullet point list containing what the user will learn during the webinar.
+    learn:
+        - Infrastructure as code concepts with Pulumi.
+        - Artifact sharing best practices using Artifactory.
+        - Utilizing Pulumi packages to standardize infrastructure.
+        - Using Artifactory to author and manage Pulumi packages.
+        - Consuming Pulumi packages stored in Artifactory .
```
Comment (cnunciato) [inline content/webinars/sharing-and-reusing-infrastructure/index.md:70]:
```suggestion
        - Consuming Pulumi packages stored in Artifactory.
```

File: layouts/page/superpowers.html [hunk 2]
```diff
@@ -378,6 +378,30 @@ <h3>
                     </div>
                 </li>
 
+                <!-- JFrog -->
+                <li class="md:flex w-full items-start pb-8 mb-12">
+                    <div class="mb-4 md:mb-0 md:mr-8 md:w-2/5">
+                        <a href="{{ relref . "/webinars/sharing-and-reusing-infrastructure" }}" class="rounded transition-all block border-2 border-transparent" title="Sharing and Reusing Infrastructure">
```
Comment (cnunciato) [inline layouts/page/superpowers.html:384]:
```suggestion
                        <a href="{{ relref . "/webinars/sharing-and-reusing-infrastructure" }}" class="rounded transition-all block border-2 border-transparent" title="Sharing and Reusing Infrastructure with Pulumi and Artifactory">
```

**Comment by justinvp** [comment]
@zchase, looks like the build is failing due to a lint error:

```
/home/runner/work/docs/docs/content/webinars/sharing-and-reusing-infrastructure/index.md:
57
Line File Header: Page title exceeds 60 characters.
```

File: content/webinars/sharing-and-reusing-infrastructure/index.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
+---
+# Name of the webinar.
+title: "Sharing and Reusing Infrastructure with Pulumi and Artifactory"
```
Comment (isaac-pulumi) [inline content/webinars/sharing-and-reusing-infrastructure/index.md:3]:
```suggestion
title: "Sharing & Reusing Infrastructure with Pulumi and Artifactory"
```

Comment (isaac-pulumi) [reply content/webinars/sharing-and-reusing-infrastructure/index.md:3]: over the character limit - try an ampersand.

**Pull request merged**